src/main/kotlin/org/jetbrains/bio/genome/AnnotationsConfig.kt

```diff
@@ -165,7 +169,7 @@ object AnnotationsConfig {
             try {
                 return GenomeAnnotationsConfig(
                     genomeAttrs["species"] as String,
-                    genomeAttrs["ucsc_alias"] as? String,
+                    ucscAlias,
```

Comment:
so maybe collect things into set, than convert to sorted list to grant the items order?

Reply:
OK, let's agree to a `LinkedHashSet` which is a set and also has order.

Reply:
Done, it's a `Set` now.